pkg/scheduler/ugm/queue_tracker.go

```diff
@@ -31,38 +31,141 @@ import (
+)
+
+func (qt *QueueTracker) increaseTrackedResource(queuePath string, applicationID string, trackType trackingType, usage *resources.Resource) bool {
 	log.Logger().Debug("Increasing resource usage",
+		zap.Int("tracking type", int(trackType)),
 		zap.String("queue path", queuePath),
 		zap.String("application", applicationID),
 		zap.Stringer("resource", usage))
 	if queuePath == "" || applicationID == "" || usage == nil {
-		return fmt.Errorf("mandatory parameters are missing. queuepath: %s, application id: %s, resource usage: %s",
-			queuePath, applicationID, usage.String())
+		return false
+	}
+
+	finalResourceUsage := qt.resourceUsage.Clone()
+	finalResourceUsage.AddTo(usage)
+	wildCardQuotaExceeded := false
+
+	// Are there any settings for specific user/group? If not, try wild card settings
+	if int(qt.maxRunningApps) == 0 && resources.Equals(resources.NewResource(), qt.maxResources) {
+		// Is there any wild card settings? Do we need to apply enforcement checks using wild card limit settings?
+		var config map[string]interface{}
+		if trackType == user {
+			config = m.getUserWildCardLimitsConfig(qt.queuePath)
+		} else if trackType == group {
+			config = m.getGroupWildCardLimitsConfig(qt.queuePath)
+		}
+		if config != nil {
+			var maxResources map[string]string
+			var maxApplications uint64
+			var ok bool
+			if maxApplications, ok = config[maxapplications].(uint64); !ok {
+				log.Logger().Debug("Problem in using the wild card limit max application settings.",
+					zap.String("queue path", queuePath),
+					zap.Int("tracking type", int(trackType)),
+					zap.Any("max applications ", config[maxapplications]))
+			}
+			if maxResources, ok = config[maxresources].(map[string]string); !ok {
+				log.Logger().Debug("Problem in using the wild card limit max resources settings.",
+					zap.String("queue path", queuePath),
+					zap.Int("tracking type", int(trackType)),
+					zap.Any("max applications ", config[maxresources]))
+			}
+			var maxResource *resources.Resource
+			var err error
+			if maxResource, err = resources.NewResourceFromConf(maxResources); err != nil {
+				log.Logger().Debug("Problem in using the wild card limit max resources settings.",
+					zap.String("queue path", queuePath),
+					zap.String("max resources after casting", maxResource.String()),
+					zap.Error(err))
+				return false
+			}
+			wildCardQuotaExceeded = (maxApplications != 0 && len(qt.runningApplications)+1 > int(maxApplications)) || (!resources.Equals(resources.NewResource(), maxResource) && resources.StrictlyGreaterThan(finalResourceUsage, maxResource))
+			log.Logger().Debug("using wild card limit settings as user/group specific limit settings not set",
+				zap.Int("tracking type", int(trackType)),
+				zap.String("queue path", queuePath),
+				zap.Uint64("wild card max running apps", maxApplications),
+				zap.String("wild card max resources", maxResource.String()),
+				zap.Bool("wild card quota exceeded", wildCardQuotaExceeded))
+		}
```

Comment:
return here if not passed: wildcard has been checked and no specific quota was set.